niyonkurueric/My_brand
# Ft landing pages-yq7WVfdI

**What does this PR do?**
Create the html and css of my brand landing pages 
**Description of Task to be completed?**
This is the page a user should see when they visit my band.
It should contain my Home page, about me, service, and contact
but this day the have same Changes and updates on Landing Page, Contact ,about  ,Services and responsive of that pages.
**How should this be manually tested?**
Pull from this branch or click on code and see dawnload zip and open index.html  in your browser then you will see the changes
**What are the relevant pivotal tracker stories?**
yq7WVfdI
**Screenshots**
![Screenshot from 2021-12-20 13-21-21](https://user-images.githubusercontent.com/76616593/146760008-c9d7fd1c-5b7d-43c7-b6de-82b173e450ec.png)
![Screenshot from 2021-12-20 13-22-08](https://user-images.githubusercontent.com/76616593/146762041-d593bcff-67fe-47d7-9cd6-736d869d24be.png)
